src/game.cpp

```diff
@@ -3363,10 +3363,10 @@ void Game::processClientEvents(CameraOrientation *cam, float *damage_flash)
 			//u16 damage = event.player_damage.amount;
 			//infostream<<"Player damage: "<<damage<<std::endl;
 
-			*damage_flash += 100.0;
-			*damage_flash += 8.0 * event.player_damage.amount;
+			*damage_flash += 63.0;
+			*damage_flash += 6.4 * event.player_damage.amount;
```

Comment:
This value should be limited to 200 or similar incase the player is hurt multiple times with a short interval.